<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QDialog" name="Console">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <property name="windowIcon">
   <iconset resource="FusionInventory.qrc">
    <normaloff>:/fusinv/fusinv.ico</normaloff>:/fusinv/fusinv.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Standard Output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextConsoleOut">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true">Standard Output:
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Standard Error</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextConsoleErr">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true">Standard Error:
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonOK">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="FusionInventory.qrc"/>
 </resources>
 <connections/>
</ui>
